plc4j/integrations/apache-kafka/README.md

```diff
@@ -60,7 +66,7 @@ In order to start a Kafka Connect system the following steps have to be performe
         
         bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic test --from-beginning
 
-### Start a Kafka Connect Worker (Standalone)
+### Start a Kafka Source Connect Worker (Standalone)
```

Comment:
The name of the product is called "Kafka Connect" ... as this is a Source for Kafka Connect, I'd probably name it "Kafka Connect Source Worker"